src/stories/docs/contributing.stories.mdx

```diff
@@ -1,5 +1,5 @@
 import { Meta } from '@storybook/addon-docs';
-import Contributing from '../../../CONTRIBUTING.md';
+import Contributing from '../../../CONTRIBUTING.mdx';
```

Comment:
Same comment: I don't think we should rename the top level `CONTTRIBUTING` and `README` files. Instead, I think we should import the raw markdown as suggested by the Storybook v7 [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#importing-plain-markdown-files-with-transcludemarkdown-has-changed):

> ```ts
> import { Markdown } from '@storybook/blocks';
> import ReadMe from './README.md?raw';
> 
> ...
> 
> <Markdown>{ReadMe}</Markdown>
> ```